riff

```diff
@@ -1,17 +1,22 @@
 #!/bin/bash
 
+RIFF_VERSION=0.0.1
```

Comment:
shouldn't this one be 0.0.2? (or will that be a single commit before the release?)

Reply:
I'll update that when we release

Reply:
cool